test/jdk/ProblemList-Xcomp.txt

```diff
@@ -28,5 +28,5 @@
 #############################################################################
 
 java/lang/invoke/MethodHandles/CatchExceptionTest.java          8146623 generic-all
-java/lang/management/MemoryMXBean/CollectionUsageThreshold.java 8318668 generic-all
+java/foreign/TestUpcallStress.java                              8341584 generic-all
```

Comment:
This looks odd, this line shouldn't be added with your PR. It makes history of commits wrong. Please check if it was correctly merged.

Reply:
Yeah, I messed up while resolving conflicts. I am checking how to correct this.